app/src/main/java/ch/epfl/sweng/eventmanager/ticketing/TicketingService.java

```diff
@@ -16,6 +16,10 @@
 import java.util.List;
 
 /**
+ * Represents a ticketing backend
+ * <p>
+ * FIXME: this should be an interface
```

Comment:
Is this fixed in another PR ?

Reply:
Maybe in the Fnux ticketing PR, but as I don't want to break his work I didn't want to change the way ticketing works for now. If we had more time, it would have been ideal to change this, but as we won't have the time, we'll leave it that way